docs/api/resource.rst

```diff
@@ -431,7 +433,7 @@ mapping validating the resource schema fields.
 
 Validation and conflicts behaviour is similar to creating records (``POST``).
 
-If the request header ``If-Unmodified-Since`` is provided, and if the record has
+If the request header ``If-Match`` is provided, and if the record has
 changed meanwhile, a ``412 Precondition failed`` error is returned.
```

Comment:
Maybe mention `If-None-Match: *` here, for 'PUT this here, but only if there is nothing there yet'